dev-support/ci/xml_to_md.py

```diff
@@ -0,0 +1,90 @@
+import os
+import re
+import zipfile
+import xml.etree.ElementTree as ET
+from collections import namedtuple
+from pathlib import Path
+
+Property = namedtuple('Property', ['name', 'value', 'tag', 'description'])
+
+def extract_xml_from_jar(jar_path, xml_filename):
+    xml_files = []
+    with zipfile.ZipFile(jar_path, 'r') as jar:
+        for file_info in jar.infolist():
+            if file_info.filename.endswith(xml_filename):
+                with jar.open(file_info.filename) as xml_file:
+                    xml_files.append(xml_file.read())
+    return xml_files
+
+def parse_xml_file(xml_content):
+    root = ET.fromstring(xml_content)
+    properties = {}
+    for prop in root.findall('property'):
+        name = prop.find('name').text if prop.find('name') is not None else ''
+        value = prop.find('value').text if prop.find('value') is not None else ''
+        tag = prop.find('tag').text if prop.find('tag') is not None else ''
+        description = prop.find('description').text if prop.find('description') is not None else ''
+        description = ' '.join(description.split()).strip() if description else ''
+        properties[name] = Property(name, value, tag, description)
+    return properties
+
+def generate_markdown(properties):
+    markdown = []
+    markdown.append("---\n")
+    markdown.append("title: \"Ozone configurations\"\n")
+    markdown.append("summary: Ozone configurations\n")
```

Comment:
The current website doesn't require front matter. The new website will fail the build if disallowed front matter keys are used like `title` and `summary`. I would change this to use `sidebar_label` only if the label in the docs sidebar should be different than the page title given by the first heading. `sidebar_laebl` is only used by the new website. The current website will ignore this key.